examples/config/domino_etl.yaml

```diff
@@ -42,21 +42,31 @@ etl:
     kind: ${etl.common.kind}
     model_type: ${etl.common.model_type}
 
-    decimation:
-      algo: decimate_pro  # can be one of {decimate_pro, decimate}
-      reduction: 0.0  # 0 means no decimation.
-      preserve_topology: false
-
   transformations:
-    # User can choose which transformations to use, for example,
-    # DoMINOZarrTransformation or DoMINONumpyTransformation.
+
+    # Preprocessing transformations.
+    # These take in the raw geometry, volume and surface data, and apply DoMINO specific preprocessing transformations.
+    # Variable names are piped in from the variables/drivaerml.yaml or variables/ahmedml.yaml file.
+    preprocessing:
+      _target_: examples.external_aerodynamics.domino.data_transformations.DoMINOPreprocessingTransformation
+      _convert_: all
+      decimation:
+        algo: decimate_pro  # can be one of {decimate_pro, decimate}
+        reduction: 0.0  # 0 means no decimation.
+        preserve_topology: false
+
+    # Default - Zarr transformation.
+    # This takes in the processed data and metadata, and converts it into a Zarr/NumPy write ready format.
+    # Users can choose between DoMINOZarrTransformation or DoMINONumpyTransformation.
     # NOTE: This should match the Sink class' serialization method.
+    # If you choose the NumPy transformation, please comment out the Zarr transformation, and uncomment the below NumPy transformation.
```

Comment:
I would suggest adding TODO to use Hydra facilities to avoid having to edit config file in multiple places.